### DISCUSSION TITLE
Better ways to use OTP in code

### DISCUSSION THREAD

**Discussion by aashutoshrathi** [mohitkyadav/yubico-js#7] [Q&A] [answered]
I saw how we currently take OTP in env and the commented test for verification.

I think there should be better ways to do it, basically not taking it in .env at least.
A way to take it using a prompt or something.
Any better ideas in mind @mohitkyadav?

**Comment by mohitkyadav** [comment, marked as answer]
Yes @aashutoshrathi it can be done by passing the real OTP in `npm test` as argument. Will create an issue for it. Thanks for bringing it to my attention. ❤️

**Reply by aashutoshrathi** [reply]
Sounds good!
